src/templates/react/Content/src/Navbar.fs

```diff
@@ -0,0 +1,48 @@
+namespace FableElmishReactTemplate
+
+module Navbar =
+
+  open Fable.Helpers.React
+  open Fable.Helpers.React.Props
+
+  let navButton classy href faClass txt =
+    a
+      [ ClassName (sprintf "button %s" classy)
+        Href href
+      ]
+      [ span
+          [ ClassName "icon"
+          ]
+          [ i
+              [ ClassName (sprintf "fa %s" faClass)
+              ]
+              [ ]
+          ]
+        span
+          [ ]
+          [ unbox txt ]
+      ]
+
+  let navButtons =
+    span
+      [ ClassName "nav-item"
+      ]
```

Comment:
I was using this notation because it's helping viewing the end a the current "block".
As the starting `[` is aligned with the ending `]`.

Anyway, I will try to use the "idiomatic" way for those []s.